go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/googlecloudplatform/gcsfuse/v2
 
-go 1.22
```

Comment:
Wouldn't this pick latest go 1.22 minor release?  Can you attach reference to the CodeQL warning about incorrect Go Toolchain version to the PR description.

Reply:
Done